dns/dyndns/src/etc/inc/plugins.inc.d/dyndns/phpDynDNS.inc

```diff
@@ -597,15 +597,15 @@ class updatedns
             case 'he-net':
             case 'he-net-v6':
                 $server = "https://dyn.dns.he.net/nic/update?";
-                curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
+                curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, true);
                 curl_setopt($ch, CURLOPT_IPRESOLVE, CURL_IPRESOLVE_V4);
```

Comment:
Small unrelated question: Should this be removed? It updates IPv4 and IPv6 cases but passes the IP address so it doesn't really matter which one is used to connect and if IPv4 is broken IPv6 doesn't update. Right? :)

Reply:
You're right, doesn't really matter if it resolves to IPv4 or IPv6 since the IP is explicitly set in the request, just tested it and it works. I'll remove the line and update the pull request (and bump the plugin version).